tests/unit/remote/ssh/test_ssh.py

```diff
@@ -38,7 +37,6 @@ def test_url(self):
         self.assertEqual(remote.user, user)
         self.assertEqual(remote.host, host)
         self.assertEqual(remote.port, port)
-        self.assertEqual(remote.prefix, path)
```

Comment:
Is this needed?

Reply:
My bad. It was wrong when I rebased with old version of code

Reply:
DONE